packages/mwp-app-render/src/components/uxcapture/UXCaptureEventMark.jsx

```diff
@@ -0,0 +1,30 @@
+// @flow
+import * as React from 'react';
+
+type onLoadHTMLElements =
+	| React.Element<'iframe'>
+	| React.Element<'img'>
+	| React.Element<'input'>;
+
+type Props = {
+	mark: string,
+	children: onLoadHTMLElements,
+};
+
+/**
+ * takes only a *single* child element
+ * and injects event based UX.mark() call into the child
+ *
+ * @see example https://github.com/meetup/ux-capture#image-elements
+ */
+export default ({ mark, children }: Props) => {
+	// if child has on onLoad prop
+	// don't do anything
+	if (children.props.onLoad) {
```

Comment:
How come we want to ignore onLoad rather than merging it here?

Reply:
Path of least resistance? :)   TBH, I'm not sure any of us fully understand other instrumentation use cases yet to be able to predict implementation details. As Mike mentioned, we may way to make this an image specific tag (i.e., `UXCaptureImageEventMark`) and or other transformations.

Reply:
Oops, submitted that too soon. Until we have a use case where we need to support an image tag with an additional onLoad prop, I just chose to ignore it. I could merge here too, but not sure if it's necessary.